src/SurveyService.Models.Dto/Requests/Group/CreateGroupRequest.cs

```diff
@@ -0,0 +1,13 @@
+using LT.DigitalOffice.SurveyService.Models.Dto.Requests.Question;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.SurveyService.Models.Dto.Requests.Group;
+
+public record CreateGroupRequest
+{
+  public string Subject { get; set; }
```

Comment:
add requred attribute 
this attribute guarantees that this property will not contain null

Reply:
Done